Makefile

```diff
@@ -37,16 +37,42 @@ clean:
 	$(MAKE) -C man clean
 	$(MAKE) -C src clean
 
+INSTALL=install
+INSTALL_PROGRAM=$(INSTALL)
+INSTALL_DATA=$(INSTALL) -m 644
+
+prefix = /usr/local
+exec_prefix = $(prefix)
+bindir = $(exec_prefix)/bin
+datarootdir = $(prefix)/share
+mandir = $(datarootdir)/man
```

Comment:
Would be great to have these be env-able too, as for pkgsrc, mandir is prefix/man, not prefix/share/man.  Probably on other places too it varies.

Reply:
Agree, and that was my intention. As far as I can tell, these are env-able (can easily be converted to uppercase, too, if needed).